src/construct-comment-utils.ts

```diff
@@ -27,11 +27,11 @@ const getCoverageDifferenceEmoji = (coverageDifference: number): string => {
 };
 
 const getCoverageIncreaseOrDecreaseSign = (difference: number): string => {
-  if (difference > 0) {
-    return `+${difference.toString()}%`;
+  if (difference>0) {
+    return `+${difference.toFixed(2).toString()}%`;
   }
 
-  return `${difference.toString()}%`;
+  return `${difference.toFixed(2).toString()}%`;
```

Comment:
```suggestion
  return `-${difference.toFixed(2).toString()}%`;
```
Should we maybe include the `minus` sign for when the difference is negative?

Reply:
agree ^

Reply:
this function is old. It is being replace in another pr

Reply:
The difference will already have a minus sign if it's negative